src/additional/nit_pis_pasepBR.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * NIT = Número de Identificação do Trabalhador (Worker Identification Number)
+ * PIS = Programa de Integração Social (Social Integration Program)
+ * Pasep = Programa de Formação do Patrimônio do Servidor Público (Civil Service Asset Formation Program)
+ * More information on http://pt.wikipedia.org/wiki/PIS/PASEP_(contribui%C3%A7%C3%A3o)
+ *
+ * Accepted formats:
+ * 99999999999
+ * 999.9999.999-99
+ */
+$.validator.addMethod("nit_pis_pasepBR", function(numero_pis) {
+	var pesos = "3298765432", resto = 0, i;
+	if (!(/^\d{3}.\d{4}.\d{3}-\d{1}$|^\d{11}$/.test(numero_pis))) {
+		return false;
+	}
+	numero_pis = numero_pis.replace(/[^\d]+/g, "");
```

Comment:
I don't like the idea of using variable names in Portuguese. The code and other variables are already in English, so that should be followed in order the maintainability.